### PR TITLE
XW-1171 send dummy details to curated main page when article doesn't exist

### PR DESCRIPTION
**Pull request by bognix**
https://wikia-inc.atlassian.net/browse/XW-1171

related PRs:
- prod@https://github.com/Wikia/app/pull/9803
- preview@https://github.com/Wikia/app/pull/9804

### CODE REVIEW

File: extensions/wikia/MercuryApi/MercuryApiController.class.php [hunk 1]
```diff
@@ -401,6 +401,7 @@ public function getPage() {
 			$title = $this->getTitleFromRequest();
 			$data = [ ];
 
+			$title->getText();
```
Comment (rogatty) [inline extensions/wikia/MercuryApi/MercuryApiController.class.php]: What's that for?

Comment (bognix) [inline extensions/wikia/MercuryApi/MercuryApiController.class.php]: accident

**Comment by tomasznapieralski** [comment]
LGTM

**Comment by hakubo** [comment]
LGTM

**Comment by rogatty** [comment]
LGTM

**Pull request merged**